addons/project_todo/views/project_task_views.xml

```diff
@@ -142,6 +142,7 @@
                         name="display_name"
                         string="To-do Title"
                         placeholder="e.g. Send Invitations"
+                        required="1"
```

Comment:
The issue is occurs due to we got `diplay_name` as False, and we try match pattern with this empty value.
So instead the require the `display_name` at view good to set fall-back value as empty string in match.

Reply:
@maan-odoo we could skip the process if display_name is falsy instead of setting an empty string to try to match something even if we know we will get nothing.

But we still need to make the display_name required in the view.